content/docs/user-guide/gems/reference/aws/aws-gamelift/build-packaging-for-windows.md

```diff
@@ -33,11 +39,34 @@ You must create a separate installation folder to copy over the required assets,
 
     -   All the `*.exe` and `*.dll` files from `o3de/build/windows_vs2019/bin/profile`.
     -   The `Registry` folder from `o3de/build/windows_vs2019/bin/profile/Registry/`.
-    -   The `VC_redist.x64.exe` file from `o3de/Tools/Redistributables/Visual Studio 2015-2019`.
   
-3. Copy all of the files in your project's cache folder, `<project folder>/Cache/pc`, to the `<package base folder>/assets/pc` folder.
+3. Copy all of the files (loose assets) in your project's cache folder, `<project folder>/Cache/pc`, to the `<package base folder>/assets/pc` folder.
+
+4. Copy the `VC_redist.x64.exe` file from `o3de/Tools/Redistributables/Visual Studio 2015-2019` to the package base folder.
+
+5. Create a build install script called `install.bat` in the package base folder and add the following content to the file. For more information about the build install script, please check [Upload a custom server build to GameLift](https://docs.aws.amazon.com/gamelift/latest/developerguide/gamelift-build-cli-uploading.html)
+
+    ```bash
+    VC_redist.x64.exe /q
+    ```
+
+### Release build
+
+1. Create the required subfolders in your installation folder. The launcher looks for fixed subfolder locations to find assets and registry files, so the installation folder must have the following structure:
+
+   - `<package base folder>/bin`
+   - `<package base folder>/assets`
+   - `<package base folder>/assets/pc`
+
+2. Copy the following files to the `/bin` folder:
+
+    -   All the `*.exe` and `*.dll` files from `o3de/build/windows_vs2019/bin/profile`.
```

Comment:
Out of curiosity, why don't they build the release configuration and copy binaries from the bin/release folder?